src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -8051,7 +8051,7 @@ private BoundExpression BindIndexedPropertyAccess(SyntaxNode syntax, BoundExpres
                     argIsIndex
                         ? ErrorCode.ERR_ImplicitIndexIndexerWithName
                         : ErrorCode.ERR_ImplicitRangeIndexerWithName,
-                    arguments.Names[0].GetLocation());
+                    arguments.Names[0].GetValueOrDefault().Location);
```

Comment:
Is a `null` location what we want for the `OrDefault` case, as opposed to `Location.None`?

Reply:
This is a move. Changed in:

https://github.com/dotnet/roslyn/pull/52422/files#diff-94aa95e94cfdcc70e3ae5562a1fa1da77438bdb4c5499304caac8dc8e0133500R8015

Reply:
Thanks